### PR TITLE
move to new grid in index page

### PR DESCRIPTION
**Pull request by erizocosmico**
New grid and adapt all things in the index (without the required changes) to it.

### CODE REVIEW

File: src/sass/pages/index.scss [hunk 1]
```diff
@@ -196,7 +197,7 @@
   }
 
   &__content {
-    @include columns(8);
+    @include columns(10);
```
Comment (dpordomingo) [inline src/sass/pages/index.scss]: reviewing the new grid, the `roadmap__content` seems to be 12 columns width; isn't it?
https://drive.google.com/file/d/0B9dw17GoC_XKR2QwY1VMVkVjNXc/view

File: src/sass/shared/components/header.scss [hunk 2]
```diff
@@ -61,7 +61,7 @@
   }
 
   ul.topbar__submenu {
-    @include width(3);
+    @include width(3.3);
```
Comment (dpordomingo) [inline src/sass/shared/components/header.scss]: 3.3 ??

File: src/sass/pages/index.scss (hunk 1 above)
Comment (erizocosmico) [inline src/sass/pages/index.scss]: Yes, but this does not apply the redesign, just makes everything look ok with the new grid.

File: src/sass/shared/components/header.scss (hunk 2 above)
Comment (erizocosmico) [inline src/sass/shared/components/header.scss]: Same as above

**Pull request merged**